garden-backend-service/src/api/routes/doi.py

```diff
@@ -32,9 +38,44 @@ async def mint_draft_doi(
     return {"doi": result.data.attributes.doi}
 
 
-@router.put("/", status_code=status.HTTP_200_OK)
+@router.put("", status_code=status.HTTP_200_OK)
 async def update_datacite(
-    body: datacite.Doi,
+    body: datacite.Doi = Body(
+        examples=[
```

Comment:
these examples should show up in the interactive docs for testing w/ more realistic payloads

Reply:
as opposed to the automatically generated datacite examples, which were overkill